api/pkg/kafka/config.go

```diff
@@ -0,0 +1,18 @@
+package kafka
+
+// Serialization format
+type SerializationFormat string
+
+const (
+	Protobuf SerializationFormat = "protobuf"
+	JSON     SerializationFormat = "json"
+)
+
+// Kafka configuration
+type Config struct {
```

Comment:
Why do we have `kafka.Config` and `config.KafkaConfig`. Can we remove the duplicate?

Reply:
initially only `kafka.Config` that existed, but due to that package relies on confluent kafka go, librdkafka must be installed also in merlin side which is unnecessary. Hence I created the new one `KafkaConfig`